bugfix: Nova Station #13 Patch

PiroMage opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание

<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->
1) Пододвинуты сканеры атмосферы у выходов в космос на более удобные места, чтобы их было видно и их не перекрывала моделька аирлок контроллеров. 
2) Переставлены вещи в оружейной брига для большей удобности.
## Ссылка на предложение/Причина создания ПР
Подготовка к релизу.
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений

<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->
![image](https://github.com/user-attachments/assets/67a27260-8734-4796-bf7e-f25b4b800c74)
![image](https://github.com/user-attachments/assets/ff890076-0d72-43e5-9d02-6d08c6adf80e)
![image](https://github.com/user-attachments/assets/2b9dd0cc-b24d-47eb-8ed1-26f0e303f015)
![image](https://github.com/user-attachments/assets/6e6ea022-828f-41a5-a9b9-84b8b1c0a16d)

## Тесты

<!-- Как вы тестировали свой код. Ревьюеру будет проще, если он будет знать какие действия вы совершали, проверяя свой ПР. -->

PiroMage committed:
Мелкие правки в карту

1) Пододвинуты сканеры атмосферы у выходов в космос на более удобные места, чтобы их было видно и их не перекрывала моделька аирлок контроллеров.
2) Переставлены вещи в оружейной брига для большей удобности.

PiroMage committed:
Дополнение

1) Добавлен скрытый сейф с соусами под пол в старую кухню.
2) Добавлен скрытый сейф с одной пачкой семян мясной пшеницы под пол в старую ботанику.

PiroMage committed:
Дополнение

1) Убраны 4 пары боевых перчаток из оружейной брига.
2) Пододвинуты некоторые сенсоры атмоса для большей красоты.
3) Добавлен недостающий сканер атмоса у выхода в космос полигона рнд.

PiroMage committed:
Правка зоны саттелита

Изменены зоны на саттелите в телепортной и комнате телекомов на разные.

PiroMage committed:
Благодарственные Таблички

Добавлены благодарственные таблички за помощь и поддержку в создании Новы.

PiroMage committed:
Дополнение старой ботаники

1) Добавлен ботанический пистолет в старую ботанику с удаленным альфа-режимом стрельбы.
2) Добавлена специальная батарейка для старого ботанического пистолета без самозаряда.

PiroMage committed:
Обновление спрайта ботанического пистолета

Изменение спрайта ботанического пистолета в старой ботанике на старый.

Daeberdir commented on code/modules/projectiles/guns/energy/special.dm:
```diff
@@ -68,6 +68,7 @@
 	selfcharge = TRUE
+	var/emagged = FALSE
 
 /obj/item/gun/energy/floragun/emag_act(mob/user)
 	. = ..()
```
```suggestion
/obj/item/gun/energy/floragun/emag_act(mob/user)
	. = ..()

	if(emagged)
		return

	if(user)
		balloon_alert(user, "протоколы защиты сняты!")

	emagged = TRUE
```

Daeberdir committed:
Completed floralgun emagging.

PiroMage committed:
Обновление кода карты

Обновлен код карты, добавлена вебмапа и добавлена карта в мап-воут.

PiroMage committed:
Merge branch 'bugfix-Nova-Station-13' of https://github.com/PiroMage/Paradise into bugfix-Nova-Station-13

PiroMage committed:
Благодарственная табличка таяры

1) Добавлена новая золотая табличка с лицом Таяры.
2) Изменены благодарственные таблички на мостике Новы.

PiroMage committed:
Описание ботанического пистолета

Обновление описания и справки старого ботанического пистолета в сейфе старой ботаники.

PiroMage committed:
Локализация пистолета ботанического

перевод описания и справки ботанического пистолета в старой ботанике

PiroMage committed:
Правка кода описания

Правка кода описания ботанического пистолета

PiroMage committed:
Убран лишний ботанический пистолет

Daeberdir commented:
@Den109G аппрув бригаду.

Daeberdir commented on code/modules/map_fluff/nova.dm:
```diff
@@ -15,5 +15,4 @@
 	starsys_name  = "Epsilon Eridani"
-	admin_only = TRUE
-	webmap_url = null //Nanomap and webmap will be added later.
+	webmap_url = "https://webmap.affectedarc07.co.uk/maps/ss1984/nova/"
```
```suggestion
	admin_only = TRUE
	webmap_url = "https://webmap.affectedarc07.co.uk/maps/ss1984/nova/"
```
Better with separated PR.

Daeberdir committed:
Update code/modules/map_fluff/nova.dm

Den109G reviewed (approved):
Выглядит нормально.
